utils/setup.cfg

```diff
@@ -3,3 +3,16 @@
 # 3. If at all possible, it is good practice to do this. If you cannot, you
 # will need to generate wheels for each Python version that you support.
 universal=1
+
+[nosetests]
+tests=../,../roles/openshift_master_facts/test/,test/
```

Comment:
@detiber hi there. When rebasing a local branch I noticed you removed `/setup.cfg` and moved the contents here, however this means that running `nosetests` from the repository root doesn't use the config file and run just 2 tests. Why did you move it? Do we expect developers to `cd utils` to run the tests, or maybe I'm missing something in my local setup?

```console
openshift-ansible$ nosetests
..
----------------------------------------------------------------------
Ran 2 tests in 0.040s

OK
openshift-ansible$ (cd utils && nosetests)
Usage: nosetests [options]

nosetests: error: Error reading config file 'setup.cfg': no such option 'with_coverage'
```

(looks like all options should have a `-` and not a `_`, I can fix that)

```console
openshift-ansible$ vim utils/setup.cfg   # replace _ with -
openshift-ansible$ (cd utils && nosetests)
...
Verify is_valid_hostname can detect None or too-long hostnames ... ok
Verify is_valid_hostname can parse regular hostnames ... ok

Name                             Stmts   Miss  Cover
----------------------------------------------------
ooinstall.py                         0      0   100%
ooinstall/cli_installer.py         602    169    72%
ooinstall/oo_config.py             262     68    74%
ooinstall/openshift_ansible.py     203     84    59%
ooinstall/utils.py                  14      0   100%
ooinstall/variants.py               37      1    97%
----------------------------------------------------
TOTAL                             1118    322    71%
----------------------------------------------------------------------
Ran 117 tests in 0.715s

OK